internal/controller/utils.go

```diff
@@ -77,7 +77,10 @@ const (
 	KindApisixConsumer     = "ApisixConsumer"
 )
 
-const defaultIngressClassAnnotation = "ingressclass.kubernetes.io/is-default-class"
+const (
+	defaultIngressClassAnnotation   = "ingressclass.kubernetes.io/is-default-class"
+	gatewayProxyNamespaceAnnotation = "apisix.apache.org/gatewayproxy-namespace"
```

Comment:
I think `apisix.apache.org/parametersNamespace` is more suitable and versatile.
what do you think?